docs/managed-identity-mount.md

```diff
@@ -0,0 +1,107 @@
+# Mount Azure SMB file share with managed identity
+ - Feature status: Preview
```

Comment:
Here it is mentioned as `Preview` feature but this feature is enabled by default in [1.34](https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/57f3c7d702673f00aca57a0f441cef37c6eb3add/charts/v1.34.1/azurefile-csi-driver/values.yaml#L130) release of files, should we need to update it to false by default (or) should we change here as supported?

Reply:
there is a new parameter(`mountWithManagedIdentity`) to decide whether mount with managed identity or not, that means if user does not use that parameter, it won't be enabled, so that's ok